src/main/java/com/phonebookservice/exception/PhoneBookControllerException.java

```diff
@@ -0,0 +1,14 @@
+package main.java.com.phonebookservice.exception;
+
+public class PhoneBookControllerException extends RuntimeException {
+
+    /**
+     * Exception class for phonebook.
+     *
+     * @param message the error message
+     */
+    public PhoneBookControllerException(final String message) {
```

Comment:
How the dispatcher will know the error code 400 or 404 or 403 ? 

Think about this and solve it